python/vegafusion/tests/test_transformed_data.py

```diff
@@ -221,10 +221,9 @@ def test_transformed_data_exclude():
 
 
 @pytest.mark.skipif(pa_major_minor < (11, 0), reason="pyarrow 11+ required")
-@pytest.mark.parametrize("connection", get_connections())
-def test_gh_286(connection):
+def test_gh_286():
```

Comment:
One thing lost in this PR is the ability to query DataFrame Interchange Protocol objects with the DuckDB connection. This was needed (under the current architecture) to avoid converting to arrow up front.